webdriver/tests/bidi/browsing_context/__init__.py

```diff
@@ -18,6 +18,7 @@ def assert_browsing_context(
     parent=None,
     url=None,
     user_context="default",
+    originalOpener=None
```

Comment:
In Python we don't use camel-case.

```suggestion
    original_opener=None
```